crates/services/consensus_module/poa/src/verifier.rs

```diff
@@ -63,13 +56,10 @@ pub fn verify_block_fields<D: Database>(
         "The `da_height` of the next block can't be lower"
     );
 
-    // Skip the verification of the time if it is possible to produce blocks manually.
-    if !config.enabled_manual_blocks {
-        ensure!(
-            header.time() >= prev_header.time(),
-            "The `time` of the next block can't be lower"
-        );
-    }
+    ensure!(
+        header.time() >= prev_header.time(),
+        "The `time` of the next block can't be lower"
+    );
```

Comment:
The manual block production increases the time monotonically, so we can remove this check=)

Reply:
even if no time interval is specified?

Reply:
Yes, the time always is `new >= prev`